src/lib.rs

```diff
@@ -81,7 +81,11 @@ impl Bibiography {
         bib_content
     }
 
-    fn generate_bibliography_html(bibliography: &HashMap<String, BibItem>) -> String {
+    fn generate_bibliography_html(
```

Comment:
As this function has already been modified, maybe adding couple of simple tests to check this function would be nice; one filtering and the other with no filter. I'll add another issue for me to add more test in other parts of the code.

Reply:
Done.